data/stories.yml

```diff
@@ -118,6 +118,22 @@ stories:
   - intent: deny
   - action: utter_cheer_up
 
+- story: happy Health_Care_Centre
+  steps:
+  - intent: Health_Care_Centre
+  - action: utter_go_to_health_care_Centre
+  - action: utter_did_that_help
+  - intent: affirm
+  - action: utter_happy
+
+- story: sad Health_Care_Centre
+  steps:
+  - intent: Health_Care_Centre
+  - action: utter_go_to_health_care_Centre
```

Comment:
hey here action name is wrong u have defined utter_go_to_health_care_center in domain but here u have utter_go_to_health_care_Centere

Reply:
I didn't notice as it seemed to be working fine, it's probably case insensitive. I will fix this in the next PR.

Reply:
if u try training the bot u will get an error

Reply:
I remember now ,I had used "replaceAll" in this file for correcting a typo. Forgot to retrain and test. Will fix this in the next PR.